From af695109e83085441d95e65d5e7795681b303500 Mon Sep 17 00:00:00 2001
From: Luis Henriques <luis@igalia.com>
Date: Mon, 29 Jun 2026 16:45:54 +0100
Subject: [PATCH 01414/15333] posix_acl: remove useless code

This is just a trivial clean-up: it removes an unnecessary return branch.

Signed-off-by: Luis Henriques <luis@igalia.com>
Link: https://patch.msgid.link/20260629154554.29093-1-luis@igalia.com
Signed-off-by: Christian Brauner (Amutable) <brauner@kernel.org>
---
 fs/posix_acl.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/fs/posix_acl.c b/fs/posix_acl.c
index b4bfe4ddf64e..6df5de23aac4 100644
--- a/fs/posix_acl.c
+++ b/fs/posix_acl.c
@@ -740,8 +740,6 @@ static int posix_acl_fix_xattr_common(const void *value, size_t size)
 	count = posix_acl_xattr_count(size);
 	if (count < 0)
 		return -EINVAL;
-	if (count == 0)
-		return 0;
 
 	return count;
 }
-- 
2.55.0

